tests/integration/targets/win_dns_zone/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+# Defines the test to run, possible values: 'standalone' or 'activedirectory'
+# 'standalone' installs a mock Windows DNS server and runs tests that only
+# include file backed DNS zones.
+# 'activedirectory' installs a mock AD/DNS server and runs tests that include
+# AD integrated and file backed DNS zones.
+win_dns_zone_test_type: standalone
```

Comment:
Does this mean the `standalone` tests only need the DNS feature to be installed and not a fully fledged AD server? Do you have a rough time as to how that takes to install in CI? If it's not too long we can at least run that.

Sorry for the mixed messages I thought before the tests required AD to be installed as well as DNS where the former cna take time we don't necessarily have in CI.